lib/connection-parameters.js

```diff
@@ -88,6 +89,9 @@ ConnectionParameters.prototype.getLibpqConnectionString = function(cb) {
   if(this.database) {
     params.push("dbname='" + this.database + "'");
   }
+  if(this.replication) {
+    params.push("replication='" + (this.replication === true ? "true" : this.replication) + "'");
```

Comment:
Since `true` stringifies to `"true"`, this can be `"replication='" + this.replication + "'"`.